web/src/Example.tsx

```diff
@@ -1,7 +1,7 @@
 /*
 
-  This is the electrified, end point of the example code
-  you should get to at the end of the Quickstart guide.
+  This is the end point of the example code that you should
```

Comment:
I don't think this comment belongs here. The web example doesn't need to reference the quickstart example. The quickstart already contains the final state code.

Reply:
Agree